routes/user.playlist.js

```diff
@@ -360,6 +360,19 @@ function generateSlideByStanza(res, songs2d, playlistName, stanzasPerSlide) {
             });
             titleMargin += 100;
         })
+        pObj = slide.addText('Powered by Glorify.cc', {
```

Comment:
Pull out the text into a variable shared between this and the next section of code that uses it.